client/admin.go

```diff
@@ -129,7 +129,7 @@ func InitMap(ctx context.Context, tree *trillian.Tree, mapClient trillian.Trilli
 				Revision: 0,
 			})
 		return err
-	})
+	}, codes.FailedPrecondition)
```

Comment:
Is "map root unavailable" the only possible `FailedPrecondition` for this RPC?

Reply:
@AlCutter -?

Reply:
I don't know, could be others if the `treeID` is unknown etc?